RELEASE_NOTES.md

```diff
@@ -1,29 +1,5 @@
 # Frequenz Repository Configuration Release Notes
 
-## Summary
-
-<!-- Here goes a general summary of what this release is about -->
-
-## Upgrading
```

Comment:
Yes.

Reply:
but you're keeping the "Bug fixes" heading.  And in the SDK you're keeping all the headings.  Maybe nice to follow a more predictable pattern.

Reply:
and bug fixes has an entry under it that wasn't cleaned up.  is that supposed to be there?

Reply:
I always try to remove all empty headings, if that's not the case, it's a bug. I'm keeping the bug fixes header because it is not empty. I'm starting to think you believe this is a release notes cleaning **after** a release, but this one is cleaning the notes to **make** a release after it is merged.

Reply:
oh I see, I thought it was after.  Then it makes sense.